kuznetsov/quizzes/src/main/java/com/kuznetsov/entities/QuestionsEntity.java

```diff
@@ -0,0 +1,79 @@
+package com.kuznetsov.entities;
+
+import org.hibernate.annotations.NaturalId;
+
+import javax.persistence.*;
+import java.util.Objects;
+
+@Entity
+@Table(name = "questions", schema = "quiz")
+public class QuestionsEntity {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int id;
+    @NaturalId
```

Comment:
Bad idea twice
1. ThemeId is not natural id
2. In Hibernate Question must contain Theme, not only id